### PR TITLE
 Sites hints - Add system

### PR DESCRIPTION
**Pull request by veteran29**
Depends on #161

### CODE REVIEW

File: game/functions/systems/sites_hints/client/fn_sites_hints_glintJob.sqf [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    Example(s):
+        call vgm_c_fnc_sites_hints_glintJob;
+ */
+
+#define RADIUS_MAX 200
+#define RADIUS_MIN 100
+#define OBJECTS_MAX 50
+
+private _interval = linearConversion [0, 1, vgm_c_skill_investigate_intensity, GLINT_JOB_INT_MAX, GLINT_JOB_INT_MIN];
+private _radius = linearConversion [0, 1, vgm_c_skill_investigate_intensity, RADIUS_MAX, RADIUS_MIN];
```
Comment (Spoffy) [inline game/functions/systems/sites_hints/client/fn_sites_hints_glintJob.sqf]:
```suggestion
#define RADIUS 200
#define OBJECTS_MAX 50

private _interval = linearConversion [0, 1, vgm_c_skill_investigate_intensity, GLINT_JOB_INT_MAX, GLINT_JOB_INT_MIN];
private _radius = RADIUS;
```

File: game/functions/systems/sites_hints/client/fn_sites_hints_glint.sqf [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ */
+
+#define ICON_COLOR [1, 1, 1, 0.5]
+
+params [
+    "_object",
+    ["_iterations", 1]
+];
+
+addMissionEventHandler ["Draw3d", {
```
Comment (Spoffy) [inline game/functions/systems/sites_hints/client/fn_sites_hints_glint.sqf]: All Draw3D handlers fire when alt-tabbing back into the game, making the glints effectively "queue" and all show at once

File: game/functions/systems/sites_hints/client/fn_sites_hints_initObject.sqf [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        Initialize hint object.
+
+    Parameter(s):
+        N/A
+
+    Returns:
+        Something [BOOL]
+
+    Example(s):
+        [parameter] call vgm_c_fnc_sites_hints_initObject
```
Comment (Spoffy) [inline game/functions/systems/sites_hints/client/fn_sites_hints_initObject.sqf]:
```suggestion
    Returns:
        Nothing

    Example(s):
        [allSitesObjects # 0, [[0, 0, 0]]] call vgm_c_fnc_sites_hints_initObject
```

File: game/functions/systems/sites_hints/client/fn_sites_hints_preInit.sqf [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        _pos set [2, 0.05];
+
+        _this setVectorUp surfaceNormal getPosATL _this;
+
+        (_this call BIS_fnc_getPitchBank) params ["_pitch"];
+        [_this, _pitch - 90, 0] call BIS_fnc_setPitchBank;
+
+        _this setPosATL _pos;
+    }]
+];
```
Comment (Spoffy) [inline game/functions/systems/sites_hints/client/fn_sites_hints_preInit.sqf:39]: 👍 , I like this

File: game/functions/systems/sites_hints/server/fn_sites_hints_place.sqf [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+    Description:
+        Place hint objects around the site.
+
+    Parameter(s):
+        _mission - Mission data or ID [NUMBER, HASHMAP]
+        _site - Site data [HASHMAP]
+
+    Returns:
+        Something [BOOL]
```
Comment (Spoffy) [inline game/functions/systems/sites_hints/server/fn_sites_hints_place.sqf]:
```suggestion
        Nothing
```

**Review by Spoffy — CHANGES_REQUESTED** [review]
Really happy with this, just pointed out a few small bits :)

File: game/functions/systems/sites_hints/client/fn_sites_hints_glint.sqf (hunk 2 above)
Comment (veteran29) [reply game/functions/systems/sites_hints/client/fn_sites_hints_glint.sqf]: Fixed by switching to EachFrame EH

**Review by Spoffy — APPROVED** [review]

**Pull request merged**